src/View.elm

```diff
@@ -45,8 +45,28 @@ icon name =
     i [ class <| "glyphicon glyphicon-" ++ name ] []
 
 
+statusContentView : Mastodon.Status -> Html Msg
+statusContentView status =
+    case status.spoiler_text of
+        "" ->
```

Comment:
Nope, the API always return a string. Empty string means no spoiler. 

I know.